regression/tests/studio/studio_base_test.py

```diff
@@ -23,6 +26,22 @@ class StudioBaseTestClass(WebAppTest):
     """
     def setUp(self):
         super(StudioBaseTestClass, self).setUp()
+
+        # Login to Lms first to avoid authentication problem
+        self.lms_login_page = LmsLogin(self.browser)
+        LoginHelper.login(self.lms_login_page)
+
+        self.login_page = StudioLogin(self.browser)
+        self.course_rerun_page = StudioRerun(self.browser)
+        self.studio_home_page = DashboardPageExtended(self.browser)
+        LoginHelper.login(self.login_page)
+        self.studio_home_page.visit()
+        self.studio_home_page.click_course_rerun()
+        self.course_rerun_page.wait_for_page()
+        self.course_rerun_page.add_rerun_with_run()
+        self.studio_home_page.wait_for_page()
+        self.browser.refresh()
```

Comment:
@benpatterson 
After rerun is added [this appears](http://screencast.com/t/Rxh8FTVy8H), which requires refresh.